Libraries/Text/Text.js

```diff
@@ -176,6 +177,12 @@ const Text: React.AbstractComponent<
     default: accessible,
   });
 
+  let flattenedStyle = flattenStyle(style);
+
+  if (typeof flattenedStyle?.fontWeight === 'number') {
+    flattenedStyle.fontWeight = flattenedStyle?.fontWeight.toString();
```

Comment:
```suggestion
    flattenedStyle.fontWeight = flattenedStyle.fontWeight.toString();
```

We already have null check inside `if` statement

Reply:
Thanks for the suggestions.